onnx/defs/gen_doc.py

```diff
@@ -371,10 +375,5 @@ def main(args):  # type: (Type[Args]) -> None
 
 
 if __name__ == '__main__':
-    base_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
-    docs_dir = os.path.join(base_dir, 'docs')
-
-    class Args(object):
-        output = os.path.join(docs_dir, 'Operators' + ext)
-        changelog = os.path.join(docs_dir, 'Changelog' + ext)
-    main(Args)
+    gen_doc(False)
```

Comment:
for better readability change this to 
gen_doc(is_ml=False)
gen_doc(is_ml=True)